src/disrupt.sh

```diff
@@ -1,15 +1,38 @@
 #!/bin/bash
 
-# Export all these variables for SC2155 rule
-export facthtml
-export tmpstring
+RANDOM_FACT="$(shuf -n 1 "${FACTS_SOURCE}")"
+RANDOM_FACT_HTML="$(echo "${RANDOM_FACT}" | sed 's/&/\&amp;/g; s/</\&lt;/g; s/>/\&gt;/g; s/"/\&quot;/g; s/'"'"'/\&#39;/g')"
 
-randomfacts="$(shuf -n 1 "$FACTS_SOURCE")"
-facthtml="$(echo "${randomfacts}" | sed 's/&/\&amp;/g; s/</\&lt;/g; s/>/\&gt;/g; s/"/\&quot;/g; s/'"'"'/\&#39;/g')"
-tmpstring="$(eval "echo \"$(cat "$RUN_DIR"/assets/template)\"")"
+DICE_ROLL=$(shuf -i 1-"${IMAGE_RARITY}" -n 1)
+
+MSG_TEMPLATE_IMG="${RUN_DIR}/assets/template_img"
+MSG_TEMPLATE_BASE="${RUN_DIR}/assets/template_head"
+IMG_ASSET_DIR="${RUN_DIR}/assets/img"
+
+EMAIL_ADDRESS="$1"
+TEMP_FILE=$(mktemp message.tmp.XXXXXXXXXXXXX)
+
+
+# add lucashifact
+cp "${MSG_TEMPLATE_BASE}" "${TEMP_FILE}"
+echo "${RANDOM_FACT_HTML}" >> "${TEMP_FILE}"
+echo '</p>' >> "${TEMP_FILE}"
+
+# add the image if necessary
+if [ "$DICE_ROLL" -eq 1 ]; then
+    RANDOM_IMAGE=$(find "${IMG_ASSET_DIR}" -type f | shuf -n 1)
```

Comment:
Should I add ``lucas.zip`` as part of this PR?

Regardless, we probably will need some sort of error-checking to handle the case when ``${IMG_ASSET_DIR}`` doesn't exist, or doesn't contain any images.